src/webgpu/util/texture/base.ts

```diff
@@ -134,10 +134,13 @@ export function viewDimensionsForTextureDimension(textureDimension: GPUTextureDi
   }
 }
 
-/** Reifies the optional fields of `GPUTextureDescriptor`. */
+/** Reifies the optional fields of `GPUTextureDescriptor`.
+ * MAINTENANCE_TODO: viewFormats should not be omitted here, but it seems likely that the
+ * @webgpu/types definition will have to change before we can include it again.
```

Comment:
I don't quite see the issue, what needs to change in the types? Would you change it to `viewFormats?: Iterable<GPUTextureFormat> | undefined`? (that's mostly identical though)

`Omit`ting `viewFormats` here actually seems fine to me. But I think we could alternatively add `viewFormats: []` below.